packages/analytics/src/tests/analytics.test.ts

```diff
@@ -0,0 +1,94 @@
+import { encodeDataToQueryString, getRandomId } from '../utils';
+import fixtures from './fixtures/encodeDataToQueryString';
+import { Analytics } from '..';
```

Comment:
avoid importing from index, use relative path to file

Reply:
47b5d0d8b39e7bc91d2cae330f1d339c73ec1323